shared/test/linkifiedTextView_test.js

```diff
@@ -37,7 +37,7 @@ describe("loop.shared.views.LinkifiedTextView", function() {
 
   describe("LinkifiedTextView", function() {
     function renderToMarkup(string, extraProps) {
-      return React.renderToStaticMarkup(
+      return ReactDOM.render(
```

Comment:
This isn't an equivalent change. We were rending to static markup previously, and now we're rending as an object.

For these tests, I think its best if we do test against the static markup - testing the dom tree would be complicated.

Therefore I think just for this shared test, we could include react-dom-server to do that rendering.